README.md

```diff
@@ -35,23 +35,64 @@ python -m urdf2webots.importer --input=someRobot.urdf [--output=outputFile] [--b
 The script accepts the following arguments:
   - **-h, --help**: Show the help message and exit.
   - **--input=INFILE**: Specifies the urdf file to convert.
-  - **--output=OUTFILE**: If set, specifies the path and, if ending in ".proto", name of the resulting PROTO file. The filename minus the .proto extension will be the robot name.
+  - **--output=OUTFILE**: If set, specifies the path and, if ending in ".proto", name of the resulting PROTO file. The filename minus the .proto extension will be the robot name (for PROTO conversion only).
+  - **--is-proto=True**: If set to false, the conversion will return a Webots Robot node string (default is `True`).
+  - **--robotName**: Specify the name of the robot (for Robot node string conversion only, in particular if you want to control multiple robots with external controllers).
```

Comment:
I thought about that and was wondering if there is a use case where we want to convert URDF into robot string without giving a name (maybe with another option to chose a non-external controller).

But as URDF is for ROS I do not think anyone wants this and it will also force users to give a name for URDF robots (which is required as soon as you have multiple robots).

Reply:
I don't believe there is a use case where we don't want to provide a name for a Robot. In the worst case, we can provide the default name, e.g., "robot".

Therefore, it would be nice to implement this simplification.